### DISCUSSION TITLE
Someone can help me...

### DISCUSSION THREAD

**Discussion by systemiii** [Koenkk/zigbee2mqtt#9415] [Devices]
I have a Home Assistant Core system on Raspberry Pi4 with CC2538 + CC2592 USB coordinator and Zigbee2mqtt addon. A couple of weeks ago I paired 8 Moes thermostatic valves that worked correctly. Today I turned off the system to add a USB extension cable to the coordinator in order to move it about 1 meter from the Raspberry. Since I turned on the system the 8 valves no more comunicate (signal level Always N/A) even though the light on the valve indicates that they are connected. If I try to do the repair in the Z2M log I read "device left network" and it disappears from the list but it doesn't do the interview anymore. Can anyone suggest me a solution, please?

**Comment by francisp2** [comment]
Probably a bad USB extension cable.

**Reply by systemiii** [reply]
I thought so too but then I was able to pair 12 Xiaomi sensors.

**Reply by systemiii** [reply]
Only the valves are unresponsive and I can't repair them

**Reply by systemiii** [reply]
An update. I can see LQI of valves in the network map page!
I really can't understand what's the matter
